### PR TITLE
Feature/tez dependency fix

### PR DESCRIPTION
**Pull request by deepakwadhwani**
CDAP-7288 : Removed packaging of tez api jar from CDAP

### CODE REVIEW

**Comment by deepakwadhwani** [comment]
Sagar is helping me build a cluster to test the fix now.

File: pom.xml [hunk 1]
```diff
@@ -1888,7 +1888,8 @@
                     <overWriteSnapshots>false</overWriteSnapshots>
                     <overWriteIfNewer>true</overWriteIfNewer>
                     <excludeGroupIds>org.apache.hadoop,org.apache.hbase,org.apache.hive,com.google.protobuf,\
-                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson</excludeGroupIds>
+                      org.datanucleus,asm,org.apache.spark,slf4j-log4j12,org.codehaus.jackson,\
+                      org.apache.tez</excludeGroupIds>
```
Comment (chtyim) [inline pom.xml]: Do you know what brings in tez-api dependency? It's good to find out which one and be able to exclude it from the transitive dependency if possible.

Comment (anwar6953) [reply pom.xml]: We have a direct dependency on tez-api, defined in the root pom and in cdap-explore/pom.xml.

Comment (deepakwadhwani) [reply pom.xml]: The direct dependency is introduced by  [https://github.com/caskdata/cdap/pull/6717/files#diff-600376dffeb79835ede4a0b285078036R114]

conf.set(TezConfiguration.TEZ_CLUSTER_ADDITIONAL_CLASSPATH_PREFIX, additionalClassPath);

**Comment by deepakwadhwani** [comment]
Hi Terence, 

It seems with the new line used tez jar was still making it in the rpm. Adding to the same line gets the expected result. Verified that tez jar file is not present on the cluster with the fix included. 

cd /opt/cdap/master/lib/
[deepak@cdap-7288-fix14631-1000 lib]$ ls _tez_
ls: cannot access _tez_: No such file or directory
[deepak@cdap-7288-fix14631-1000 lib]$ pwd
/opt/cdap/master/lib
[deepak@cdap-7288-fix14631-1000 lib]$

Thanks
Deepak.

**Comment by chtyim** [comment]
LGTM

**Comment by chtyim** [comment]
Please squash the commits before merging (not using the squash in Github, but squash using the git command).

**Comment by deepakwadhwani** [comment]
Finished the squash. 
Test is running now: http://builds.cask.co/browse/CDAP-DUT4947-3

**Comment by deepakwadhwani** [comment]
Develop Build and Test 
Child of CDAP-DRC4688-3 2 hours ago 2212 passed

**Pull request merged**